bazel/example/BUILD

```diff
@@ -0,0 +1,8 @@
+cc_test(
+    name = "main",
+    size = "small",
+    srcs = ["main.cc"],
+    deps = [
+        "@com_google_glog//:glog",
```

Comment:
You don't need to reference the workspace name here since you are in the same workspace

Reply:
Done. This was left over from my first attempt to have the example use com_google_glog as an external repo.

Thanks for the review!